plonky3/Cargo.toml

```diff
@@ -22,6 +22,8 @@ nightly-features = [
   "p3-mersenne-31/nightly-features",
   "p3-poseidon2/nightly-features",
 ]
+# As a guest in powdr, use accelerated machines.
+powdr-accel = []
```

Comment:
powdr-accel sounds a bit weird, why not just powdr?

Reply:
Because the thing is accelerated with powdr.

I think a feature named just `powdr` in a library of the powdr project, in a repo named powdr, is confusing.

If not `powdr-accel`, maybe `guest-in-powdr` or something like it?

Reply:
ok, then `powdr-accel` it is